trex-config/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/centos/centos:stream9
+
+ARG TREX_VERSION=2.87
+ENV TREX_VERSION ${TREX_VERSION}
+
+# install requirements
+RUN dnf -y install --nodocs git wget procps python3 vim python3-pip pciutils gettext https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && dnf clean all
+RUN dnf install -y --nodocs hostname iproute net-tools ethtool nmap iputils perf numactl sysstat htop rdma-core-devel libibverbs libibverbs-devel net-tools && dnf clean all
```

Comment:
Please consider using a single RUN stanza, with escaped new line characters.
Using two RUN stanzas, creates two layers in the image (and needs to be cleaned twice).

Reply:
DONE